src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -103,7 +103,93 @@ Sentry.init({
 });
 ```
 
-Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry. Here's an example for your server component:
+Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry.
+
+### Using a Tunneling Server
+
+Here's an example for your server component:
+
+```js {tabTitle:Express}
+// Change this to suit your needs
+const SENTRY_HOST = "oXXXXXX.ingest.sentry.io";
+const SENTRY_KNOWN_PROJECTS = ["123456"];
+
+app.post("/bugs", async (req, res) => {
```

Comment:
h: You cannot have handlers in express be async. This will mess up error handling.